### ISSUE TITLE
Unable to download frankenphp from Caddy Downloads

### ISSUE THREAD

**Issue by loalf** [dunglas/frankenphp#1085]
### What happened?

I'm trying to install frankenphp as a Caddy module, but this URL fails

```
https://caddyserver.com/api/download?os=linux&arch=amd64&p=github.com/dunglas/frankenphp/caddy
```

### Build Type

Custom (tell us more in the description)

### Worker Mode

Yes

### Operating System

GNU/Linux

### CPU Architecture

x86_64

### PHP configuration

```shell
It's not php.
```

**Comment by withinboredom** [comment]
Frankenphp is implemented as a Caddy module, but cannot be installed as a caddy module on its own. It requires a PHP installation, compiled in ZTS mode, and other dependencies. Instead, you use xcaddy on frankenphp and install caddy modules on it.

**Comment by mholt** [comment]
Yeah; sorry, our download page does not support actually building with FrankenPHP (cgo and other unusual build requirements). We'll try to make this more apparent.

**Comment by dunglas** [comment]
Not a FrankenPHP issue, confirmed by Caddy maintainer.
